pkg/apis/provisioning/v1alpha5/labels.go

```diff
@@ -43,6 +43,7 @@ var (
 	// they are not used in a context where they may be passed as argument to kubelet.
 	LabelDomainExceptions = sets.NewString(
 		"kops.k8s.io",
+		"node.kubernetes.io",
```

Comment:
https://github.com/kubernetes/kubelet/blob/master/pkg/apis/well_known_labels.go#L50

```suggestion
         v1.LabelNamespaceSuffixKubelet,
	v1.LabelNamespaceSuffixNode,	
```